### PR TITLE
chore: add update commands for public api requests

### PR DESCRIPTION
**Pull request by fraidev**
This PR is adding Update commands for the public api requests. 

This PR is needed for the add partition feature: https://github.com/infinyon/fluvio/pull/4065

### CODE REVIEW

File: crates/fluvio-sc-schema/src/objects/classic.rs [hunk 1]
```diff
@@ -412,6 +413,197 @@ mod delete_macro {
     pub(crate) use ClassicDecodingDelete;
 }
 
+/// carry from prev version for compatibility test
+mod update_macro {
+
```
Comment (sehz) [inline crates/fluvio-sc-schema/src/objects/classic.rs:418]: should not be needed.  only for supporting old protocol

**Review by sehz — CHANGES_REQUESTED** [review]
looks good.  just remove compatibility macro

File: crates/fluvio-sc-schema/src/objects/classic.rs (hunk 1 above)
Comment (fraidev) [reply crates/fluvio-sc-schema/src/objects/classic.rs]: Make sense! We don't need this. 

I removed that.

**Review by sehz — APPROVED** [review]
LGTM

**Pull request merged**